projects/online_restaurant/src/main/resources/application.properties

```diff
@@ -1,6 +1,6 @@
 bootcoding.datasource.jdbcUrl=jdbc:postgresql://localhost:5432/bootcoding_db
 bootcoding.datasource.username=postgres
-bootcoding.datasource.password=postgres
+bootcoding.datasource.password=admin@123
```

Comment:
Do not push this file if only password is modified (because it belongs to your local database)